src/main/java/thePackmaster/powers/instadeathpack/AnotherMomentPower.java

```diff
@@ -0,0 +1,57 @@
+package thePackmaster.powers.instadeathpack;
+
+import com.megacrit.cardcrawl.actions.common.ApplyPowerAction;
+import com.megacrit.cardcrawl.actions.utility.UseCardAction;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.core.AbstractCreature;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.powers.DexterityPower;
+import thePackmaster.SpireAnniversary5Mod;
+import thePackmaster.powers.AbstractPackmasterPower;
+
+
+public class AnotherMomentPower extends AbstractPackmasterPower {
+    public static final String POWER_ID = SpireAnniversary5Mod.makeID("AnotherMomentPower");
+    private static final String NAME = CardCrawlGame.languagePack.getPowerStrings(POWER_ID).NAME;
+    private static final String[] DESCRIPTIONS = CardCrawlGame.languagePack.getPowerStrings(POWER_ID).DESCRIPTIONS;
+
+    private int exhaustedThisTurn = 0;
+
+    public AnotherMomentPower(final AbstractCreature owner, int amount)
+    {
+        super(POWER_ID, NAME, PowerType.BUFF, false, owner, amount);
+    }
+
+    public void atStartOfTurn() {
+        this.exhaustedThisTurn = 0;
+    }
+
+    public void onUseCard(AbstractCard card, UseCardAction action) {
+        if (card.type == AbstractCard.CardType.SKILL) {
```

Comment:
Might've been good as a two amount power?

Reply:
eh maybe but I think it's fine as is, functions the same as echo form